src/components/request/RequestForm.tsx

```diff
@@ -40,6 +62,11 @@ const RequestForm: FC = () => {
     }
   };
   const checkForInputError = (v: string) => {
+    if (eventBased && v === MagicNumber) {
+      throw new Error(
+        "Event based expiry requests cannot be resolved with too early ie magic number"
```

Comment:
nit: should probably rather be `Event based expiry requests cannot be resolved with "too early", i.e. "magic number".`